Index: JAVATEAMPROJECT/src/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import views.DefaultFrame;\r\nimport views.DeleteDrinksPopup;\r\nimport views.DrinksManagementView;\r\nimport views.ProductListCartView;\r\n\r\nimport java.awt.*;\r\nimport javax.swing.*;\r\n\r\npublic class Main {\r\n    public static void main(String[] args) {\r\n        System.out.println(System.getProperty(\"user.dir\"));\r\n        DefaultFrame win = new DefaultFrame();\r\n        win.setSize(1920,1080);\r\n        //add는 setVisible보다 위에 있어야 됨!\r\n            win.add(new ProductListCartView(win));\r\n        win.setVisible(true);\r\n        win.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);\r\n\r\n    }\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/JAVATEAMPROJECT/src/Main.java b/JAVATEAMPROJECT/src/Main.java
--- a/JAVATEAMPROJECT/src/Main.java	(revision 5d6260add58651cc8801e3615e4e8547cdadb360)
+++ b/JAVATEAMPROJECT/src/Main.java	(date 1701019190607)
@@ -8,11 +8,10 @@
 
 public class Main {
     public static void main(String[] args) {
-        System.out.println(System.getProperty("user.dir"));
         DefaultFrame win = new DefaultFrame();
         win.setSize(1920,1080);
         //add는 setVisible보다 위에 있어야 됨!
-            win.add(new ProductListCartView(win));
+        win.add(new ProductListCartView(win));
         win.setVisible(true);
         win.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
 
