Index: JAVATEAMPROJECT/src/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import views.*;\r\n\r\nimport java.awt.*;\r\nimport javax.swing.*;\r\n\r\npublic class Main {\r\n    public static void main(String[] args) {\r\n        DefaultFrame win = new DefaultFrame();\r\n        win.setSize(1920,1080);\r\n        win.add(new RoomSettingView(win));\r\n        win.setVisible(true);\r\n        win.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);\r\n    }\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/JAVATEAMPROJECT/src/Main.java b/JAVATEAMPROJECT/src/Main.java
--- a/JAVATEAMPROJECT/src/Main.java	(revision ebe1588b7724fddc12a410c5740202385bcaf3c2)
+++ b/JAVATEAMPROJECT/src/Main.java	(date 1701099789631)
@@ -5,6 +5,10 @@
 
 public class Main {
     public static void main(String[] args) {
+        DBConnect dbConnect = new DBConnect();
+        Controller controller = new Controller(dbConnect.getConn());
+
+
         DefaultFrame win = new DefaultFrame();
         win.setSize(1920,1080);
<<<<<<< Updated upstream
         win.add(new Roo
=======
         win.add(new RoomSettingView(win));
>>>>>>> Stashed changes
